data/oecd/oecd_tariffs.ipynb

```diff
@@ -124,6 +124,7 @@
     "# Load country mapping to ISO code\n",
     "metadata_dir = \"metadata\"\n",
     "path_to_country_conversion_table = os.path.join(metadata_dir, \"country_name_conversion.csv\")\n",
+    "missing_countries = []\n",
```

Comment:
I see: you've move `missing_countries` definition up, above the `if` in order to use it in the `elif`.
 Then you need to delete it a few line down, within the `if`. Otherwise it's re-defined and confusing.